crates/consensus/src/block.rs

```diff
@@ -0,0 +1,38 @@
+//! Genesic Block Type
+
+use crate::{Header, Requests};
+use alloc::vec::Vec;
+use alloy_eips::{
+    eip2718::{Decodable2718, Encodable2718},
+    eip4895::Withdrawal,
+};
+use alloy_rlp::{Decodable, Encodable, RlpDecodable, RlpEncodable};
+
+/// Ethereum full block.
+///
+/// Withdrawals can be optionally included at the end of the RLP encoded message.
+///
+/// Taken from [reth-primitives](https://github.com/paradigmxyz/reth)
```

Comment:
can we add this as context:

https://github.com/paradigmxyz/reth/blob/3fa1e28e12cf7b05a365e93ade19a6de9e2aafd3/crates/primitives/src/block.rs#L567-L569

and reference 

https://github.com/ethereum/devp2p/blob/master/caps/eth.md#block-encoding-and-validity